caselaw/settings.py

```diff
@@ -53,6 +53,7 @@
     "django.middleware.security.SecurityMiddleware",
     "django.contrib.sessions.middleware.SessionMiddleware",
     "django.middleware.common.CommonMiddleware",
+    "django.middleware.locale.LocaleMiddleware",
```

Comment:
Maybe you should move this middleware before the common one as there is a chance that some translations are rendered from views and not from templates and then this will cause problems, but it should be after the sessions as it depends on that to infer the language.
[Reference](https://docs.djangoproject.com/en/2.2/topics/i18n/translation/#how-django-discovers-language-preference)

Reply:
Thanks a lot. I have fixed this issue